Jenkinsfile

```diff
@@ -12,6 +12,15 @@ def buildVscodeExtension(){
 	sh "npm run vscode:prepublish"
 }
 
+def buildLemMinXBinary() {
+	build job: 'NativeImage.jenkins', parameters: [
+		// This parameter only affects the Windows and macOS builds.
+		// In order to update GraalVM for rhel8, you need to modify the yaml scripts that are used to generate the rhel8 vm image.
+		string(name: 'GRAALVM_VERSION', value: '20.3.0'),
+		string(name: 'UPLOAD_LOCATION', value: UPLOAD_LOCATION)
```

Comment:
pass `publishToMarketPlace`  as `stable` parameter

Reply:
AFAIK, this isn't possible with the way I am running the NativeImage.jenkins file from the other Jenkinsfile. I read the `publishToMarketplace` parameter directly. If you know a better way to introduce a new parameter, please let me know.